.github/pull_request_template.md

```diff
@@ -4,6 +4,7 @@
    - 2️⃣ PR body: Replace 'body...' with detailed description of the change.
    - 3️⃣ category: pick one, delete the rest.
    - 4️⃣ ticket: Replace #000 with link to a GitHub issue (or 'none' if PR is trivial).
+   - 5️⃣ feature_set: pick one (or delete completely if not applicable).
```

Comment:
what are the cases of non-applicability?
maybe updating github actions would be one.

Reply:
Only `category: feature` requires `feature_set` I think

Reply:
Cool